com.github.shonumi.gbe-plus.json

```diff
@@ -0,0 +1,68 @@
+{
+    "app-id" : "com.github.shonumi.gbe-plus",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "18.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "gbe_plus",
+    "finish-args" : [
+        /* X11 + XShm access */
+        "--share=ipc", "--socket=x11",
+        /* GPU acceleration */
+        "--device=dri",
+        /* Play sounds */
+        "--socket=pulseaudio",
+        /* Get access to the files */
+        "--filesystem=host:ro",
+        "--filesystem=xdg-run/gvfs:ro"
```

Comment:
Is GVFS used by Qt?

Reply:
No, I didn't update the finish-args in that version.